Handheld/Android/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <manifest package="com.esri.arcgisruntime.opensourceapps.DSA_Handheld_Qt" xmlns:android="http://schemas.android.com/apk/res/android" android:versionName="1.1.6" android:versionCode="7" android:installLocation="auto">
-    <uses-sdk android:minSdkVersion="21" android:targetSdkVersion="29"/>
+    <uses-sdk android:minSdkVersion="26" android:targetSdkVersion="29"/>
     <!-- The following comment will be replaced upon deployment with default permissions based on the dependencies of the application.
          Remove the comment if you do not require these default permissions. -->
     <!-- %%INSERT_PERMISSIONS -->
```

Comment:
I was planning on working on the Android porting as a separate PR since the permissions and data location will need to be updated like we did in samples...